backend/QA/src/main/java/com/example/qa/dao/question/QuestionDao.java

```diff
@@ -188,6 +188,13 @@ public void update(
       throws DaoException {
     Put put = createPutWithForUpdate(date, createdAt, updatedAt, numAnswer);
 
+    /** To manage partial updates on Cosmos DB, Cosmos DB does not currently support partial
+     *  document updates*/
+    QuestionRecord question = get(date, createdAt, storage);
+    put.withValue(new TextValue(COL_NAME_TITLE, question.getTitle()))
+    .withValue(new TextValue(COL_NAME_USER, question.getUser()))

```

Comment:
format

Reply:
Thank you for the suggestion.
Document update accordingly.